Header header

uint8 pin_num
bool high_low